### DISCUSSION TITLE
Topology diagrams could not be seen at same time under the General-root topology page when use oap-9.5.0

### DISCUSSION THREAD

**Discussion by arielwei123** [apache/skywalking#11088] [Q&A]
When use oap-9.2.0,  all the topology diagrams could be seen under the general-root topology page at the same time.  When use oap-9.5.0,  topology diagram could be seen under each general-service topology page but not all seen under the general-root topology page.

9.2.0 general-root topology page,
<img width="417" alt="image" src="https://github.com/apache/skywalking/assets/135794581/d4628ecf-50b3-467f-af89-3c26824a19a9">

9.5.0
general-service topology page,
<img width="358" alt="1689246447(1)" src="https://github.com/apache/skywalking/assets/135794581/43df9c3a-57ee-4316-a401-1ab8706b73e5">
<img width="345" alt="1689246488(1)" src="https://github.com/apache/skywalking/assets/135794581/2f209e6a-3ce1-4079-ac60-28b936d52de0">

general-root topology page,
<img width="437" alt="1689246527(1)" src="https://github.com/apache/skywalking/assets/135794581/bb9580f6-d143-4ee3-a9ee-fcdbfb546ed5">

**Comment by wu-sheng** [comment]
I can see the topo still there on demo env.

**Reply by wu-sheng** [reply]
Generally, didn't your last screenshot show the topo of `general-root`? What is the issue?

**Comment by arielwei123** [comment]
Yes, the last screenshot show the topo of general-root.  I mean the topo of service 39-vt-python-agent and sonar-mysql-agent should both be seen under general-root(as 9.2.0 shown) , but only topo of service 39-vt-python-agent there.

**Reply by wu-sheng** [reply]
No, it doesn't have to be there.
If that service is not in that layer, or/and no network traffic, it wouldn't be there.
You need to debug on your own.
